store/tikv/lock_resolver.go

```diff
@@ -405,6 +413,13 @@ func (lr *LockResolver) resolveLock(bo *Backoffer, l *Lock, status TxnStatus, cl
 		if status.IsCommitted() {
 			req.ResolveLock.CommitVersion = status.CommitTS()
 		}
+		if l.TxnSize < BigTxnThreshold {
+			// Only resolve specified keys when it is a small transaction,
+			// prevent from scanning the whole region in this case.
+			tikvLockResolverCountWithResolveLockLite.Inc()
+			cleanWholeRegion = false
```

Comment:
Once `cleanWholeRegion` is set to false, it won't be set to true again?

Reply:
Yes